jen/group.go

```diff
@@ -118,8 +118,22 @@ func (g *Group) renderItems(f *File, w io.Writer) (isNull bool, err error) {
 // Render renders the Group to the provided writer.
 func (g *Group) Render(writer io.Writer) error {
 	f := NewFile("")
+	return g.RenderWithFile(writer, f)
```

Comment:
Let's combine this with the previous line:

```go
return g.RenderWithFile(writer, NewFile(""))
```

Reply:
Fixed